src/libraries/System.IO.Compression/ref/System.IO.Compression.cs

```diff
@@ -131,18 +131,30 @@ public enum ZipArchiveMode
         Create = 1,
         Update = 2,
     }
-    public sealed class ZLibCompressionOptions
+    public sealed partial class ZLibCompressionOptions
     {
-        public int CompressionLevel { get; set; }
-        public ZLibCompressionStrategy CompressionStrategy { get; set; }
+        public ZLibCompressionOptions() { }
```

Comment:
Same as BrotliCompressionOptions - fine to add it to make the GenAPI diff clean, but doesn't imply any missing functionality or breaks.